api/manifest/manifest.go

```diff
@@ -35,6 +36,10 @@ var (
 	SignalWaiter = waitForSignal
 )
 
+var RandomPort = func() int {
+	return 10000 + rand.Intn(50000)
```

Comment:
this will always generate ports in the same order, but probably not an issue for this